gradle.properties

```diff
@@ -19,7 +19,7 @@ BINTRAY_VERSION=1.8.4
 GRADLE_VERSION=5.6.4
 OPENAPI_VERSION=7.0.3
 GRADLE_PLUGIN_PUBLISH_VERSION=0.10.1
-INTELLIJ_GRADLE_PLUGIN_VERSION=0.4.16
+INTELLIJ_GRADLE_PLUGIN_VERSION=0.4.17
```

Comment:
How are we working towards this in meta? do we always target the latest stable intellij gradle plugin?

Reply:
Raquel can answer that better but the CI bot has a compatibility matrix

Reply:
Hi! :raised_hands: 

CI bot considers:
* Intellij IDEA version
* Kotlin IDEA Plugin version
* Kotlin Compiler version

However. Intellij IDEA **Gradle Plugin** version consideration is missing. 

I added a task to include it after reviewing this PR: https://github.com/arrow-kt/arrow-meta/pull/498